content/learn/getting-started/node-with-typescript.fr.md

```diff
@@ -2,7 +2,7 @@
 title: nodejs-with-typescript
 displayTitle: 'Node.js avec TypeScript'
 description: "Découvrez pourquoi TypeScript est un outil génial et apprenez à l'utiliser par vous-même."
-authors: sbielenica, AugustinMauroy
+authors: sbielenica, ovflowd, vaishnav-mk, AugustinMauroy
```

Comment:
Why was my name added here? Did I contribute to the English one? (I don't even remember)

Reply:
You are in English file ...